src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_route_table/azuredeploy.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "routeTableName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name for route table."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for resource."
```

Comment:
We haven't called out what the default is in the past for location.  Since the location is overridden by registered args, we could override the CLI params all at once if we decide to make the change.

---
In reply to: [74152823](https://github.com/Azure/azure-cli/pull/668#discussion_r74152823) [](ancestors = 74152823)